src/components/timeline/Thread.js

```diff
@@ -177,29 +170,28 @@ class ProfileThreadHeaderBar extends PureComponent<Props> {
         thread.name === 'Compositor' ||
         thread.name === 'Renderer') &&
       processType !== 'plugin';
-    const className = 'profileThreadHeaderBar';
 
     return (
       <li
-        className={'profileThreadHeaderBar' + (isSelected ? ' selected' : '')}
+        className={'timelineThread' + (isSelected ? ' selected' : '')}
         onClick={this._onLineClick}
         style={style}
       >
         <ContextMenuTrigger
-          id={'ProfileThreadHeaderContextMenu'}
+          id={'TimelineThreadContextMenu'}
           renderTag="div"
           attributes={{
             title: processDetails,
-            className: 'grippy profileThreadHeaderBarThreadLabel',
+            className: 'grippy timelineThreadLabel',
             onMouseDown: this._onLabelMouseDown,
           }}
         >
-          <h1 className="profileThreadHeaderBarThreadName">{threadName}</h1>
+          <h1 className="timelineThreadName">{threadName}</h1>
         </ContextMenuTrigger>
-        <div className="profileThreadHeaderBarThreadDetails">
+        <div className="timelineThreadDetails">
           {displayJank ? (
-            <ProfileThreadJankOverview
-              className={`${className}IntervalMarkerOverview ${className}IntervalMarkerOverviewJank`}
+            <TimelineTracingMarkersJank
+              className="timelineThreadIntervalMarkerOverview"
```

Comment:
My first feeling was that I'd like to keep an additional classname specifically for the jank line, in case we need to target it later.
But when thinking about it more, we can add it when/if we need it later. So this is OK for me.